samples/test/support/action/enterIncorrectCode.ts

```diff
@@ -3,7 +3,7 @@ import setInputField from './setInputField';
 import clickElement from './clickElement';
 
 export default async function () {
-  const code = '!incorrect!';
+  const code = '000000';
```

Comment:
I actually prefer the previous invalid code, as it indicates what happens in test. Does it cause any issue?

Reply:
Reverted